libcst/codemod/visitors/_gather_exports.py

```diff
@@ -53,6 +53,17 @@ def visit_AnnAssign(self, node: cst.AnnAssign) -> bool:
                 return True
         return False
 
+    def visit_AugAssign(self, node: cst.AugAssign) -> bool:
+        target_name = get_full_name_for_node(node.target)
+        if target_name == "__all__":
+            value = node.value
+            if isinstance(value, (cst.List, cst.Tuple)) and isinstance(
+                node.operator, cst.AddAssign
+            ):
```

Comment:
Those two if statements can be replaced as a matcher.
```suggestion
            if m.matches(node, m.AugAssign(target=m.Name("__all__"), operator=m.AddAssign, value=m.List() | m.Tuple())):
```